include/streaming_flatbuffers.h

```diff
@@ -48,13 +48,11 @@ static void readFlatBufferTableFromStream(
   std::istream& in,
   std::function<void(const T&)> tableReader) {
     size_t bufferSize;
+    in.ignore(1); // Ignores next space
```

Comment:
Hm?

Reply:
This is totally in the wrong place, fixed